src/Comment.jsx

```diff
@@ -34,7 +34,7 @@ class _Comment extends React.Component {
         this.state = {
             editMode: false,
             editedComment: null,
-            savingComment: false,
+            savingComment: props.comment && !props.comment.createdAt && !props.comment.updatedAt,
```

Comment:
What would be the case for 'creating' this component and setting savingComment to true?

Reply:
I suppose that if a comment is posted, we can immediately show it in the list without waiting for the server's response. We know its content, visibility and author by default - so we can instantiate the component right away.

Reply:
yep, that's the case. 👍